app/Http/Controllers/Api/v1/User/Controllers/GenerateTokenController.php

```diff
@@ -42,13 +44,42 @@ public function __invoke(Request $request): JsonResponse
         try {
             $user = $this->userLookup->getUpdatedUser($user, $azureUser->principal_name);
         } catch (ClientExceptionInterface) {
+            $log = new Log();
+            $log->user_id = $user->id;
+            $log->severity = '5';
+            $log->category = 'mensa';
+            $log->text = json_encode((object) [
+                'result' => "failure",
+                'data' => "Updating user failed: HTTP_BAD_GATEWAY"
+            ]);             
+            $log->text = json_encode();
```

Comment:
You are overwriting the previous text statement?

Reply:
this might be an error... couldn't test this part yet